Maven/ExtractPage.java

```diff
@@ -0,0 +1,52 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class ExtractPage {
+
+    public static String URLtoCheck="https://www.placelab.com/extract";
+    public String browser;
+    public WebDriver driver;
+
+    public ExtractPage(){
+
+    }
+
+    @BeforeTest
+    public void openBrowser(){
+        this.browser="Firefox";
+
+        if (this.browser.contains("Chrome")){
+            WebDriverManager.chromedriver().setup();
+            this.driver = new ChromeDriver();
+        } else if (this.browser.contains("Firefox")){
+            WebDriverManager.firefoxdriver().setup();
+            this.driver = new FirefoxDriver();
+        }
+        this.driver.navigate().to(URLtoCheck);
+    }
+
+    @Test
+    public void verifyLetsTalkRedButton(){
+        String expectedText = "LET'S TALK";
+        String actualText = "";
+
+        By by = new By.ByXPath("/html/body/header/div[1]/div/div/div/a/button/div");
```

Comment:
👏  for using different application element for this homework. As I've mentioned, we will generally try to make xpaths as smaller as possible and they don't need to go from the root element. 
You can iterate from some relative location using `//`. But you will learn more on this topic anyway as part of the qamp so this is good for now :)

Reply:
File updated with more content. Let me know what you think. Thanks!